Documentation/ApiOverview/ErrorAndExceptionHandling/Extending/Index.rst

```diff
@@ -8,12 +8,12 @@ How to extend the error and exception handling
 ==============================================
 
 If you want to register your own error or exception handler, simply
-include the class and insert its name into "productionExceptionHandler",
-"debugExceptionHandler" or "errorHandler"::
+include the class and insert its name into `productionExceptionHandler`,
+`debugExceptionHandler` or `errorHandler`::
 
-   $GLOBALS['TYPO3_CONF_VARS']['SYS']['errorHandler'] = 'myOwnErrorHandler';
-   $GLOBALS['TYPO3_CONF_VARS']['SYS']['debugExceptionHandler'] = 'myOwnDebugExceptionHandler';
-   $GLOBALS['TYPO3_CONF_VARS']['SYS']['productionExceptionHandler'] = 'myOwnProductionExceptionHandler';
+   $GLOBALS['TYPO3_CONF_VARS']['SYS']['errorHandler'] = 'Vendor\Ext\Error\MyOwnErrorHandler';
```

Comment:
I would advise to use the fully qualified class name with `::class` as is done in typo3/sysext/core/Configuration/DefaultConfiguration.php. I will push another change.